pkg/storage/cache/factory/factory.go

```diff
@@ -0,0 +1,36 @@
+package factory
```

Comment:
Instead of "factory", maybe use "backend" instead

Reply:
Moved factory to backend, renamed cachedbfactory to cacheBackend